vllm/config/speculative.py

```diff
@@ -224,7 +219,7 @@ def __post_init__(self):
         # can not be detected, it will be considered as the "draft_model" by
         # default.
 
-        if self.method in MTP_MODEL_TYPES:
+        if self.method in get_args(MTPModelTypes):
```

Comment:
This warning is printing erroneously since "mtp" was added to "MTPModelTypes" 
```suggestion
        if self.method in get_args(MTPModelTypes) and self.method != "mtp":
```

Reply:
ok，i will fix this